includes/scrub-options.php

```diff
@@ -90,6 +91,11 @@ function( $v ) use ( $modules_to_disable ) {
 		}
 	}
 
+	// Disable AutomateWoo workflows and clear the queue
+	$wpdb->query( "UPDATE $wpdb->posts SET post_status = 'aw-disabled' WHERE post_type = 'aw_workflow' AND post_status = 'publish'" );
+	$wpdb->query( "DELETE FROM {$wpdb->prefix}automatewoo_queue" );
+	$wpdb->query( "DELETE FROM {$wpdb->prefix}automatewoo_queue_meta" );
+
```

Comment:
Should the `wp_actionscheduler_actions` table be cleared and/or have status updated, as well? I noticed that was needed in the related incident )p4Kr4c-dAP-p2) to stop actions that were in progress (and may be in progress at the time of cloning or by the time Safety Net is installed).

This SQL was referenced as a way to disable the actions in the AW tables:
`Update `wp_actionscheduler_actions` set status = 'done' WHERE status = 'pending' and hook like '%automatewoo%';`

Reply:
Good call, I'll add that. Thanks Mike!